lms/djangoapps/courseware/tests/test_microsites.py

```diff
@@ -322,8 +322,7 @@ def test_paid_course_registration(self):
         url = reverse('about_course', args=[text_type(self.course_with_visibility.id)])
         resp = self.client.get(url, HTTP_HOST=settings.MICROSITE_TEST_HOSTNAME)
         self.assertEqual(resp.status_code, 200)
-        self.assertNotIn(u"Enroll in {}".format(
-            self.course_with_visibility.id.course),
+        self.assertNotIn(u"Enroll Now",
```

Comment:
The style linter is complaining about this line. Please change it to:

```python
self.assertNotIn(u"Enroll Now", resp.content.decode(resp.charset))
```

FWIW, the following are also acceptable permutations when lines get longer:

```python
self.assertNotIn(
    u"Enroll Now", resp.content.decode(resp.charset)
)
```

```python
self.assertNotIn(
    u"Enroll Now",
    resp.content.decode(resp.charset),
)
```